packages/react-native-renderer/src/ReactNativeHostConfig.js

```diff
@@ -488,6 +488,10 @@ export function clearContainer(container: Container): void {
   // UIManager does not expose a "remove all" type method.
 }
 
+export function clearInstanceHandle(instance: Instance) {
+  uncacheFiberNode(instance);
```

Comment:
You'll need to import `uncacheFiberNode` though, right?

Reply:
No, it's already defined in this file. It turns out we already do this so it's unnecessary.

Reply:
Ah, makes sense to your other comment.